project/Paradox.scala

```diff
@@ -35,8 +35,8 @@ object Paradox {
       "extref.platform-guide.base_url" -> "https://developer.lightbend.com/docs/akka-platform-guide/%s",
       "extref.wikipedia.base_url" -> "https://en.wikipedia.org/wiki/%s",
       "extref.github.base_url" -> (GitHub.url(version.value) + "/%s"), // for links to our sources
-      "extref.samples.base_url" -> "https://developer.lightbend.com/start/?group=akka&amp;project=%s",
-      "extref.ecs.base_url" -> "https://example.lightbend.com/v1/download/%s",
+      "extref.samples.base_url" -> s"$pekkoBaseURL/docs/pekko-samples/current/%s",
+      "extref.akka-samples.base_url" -> "https://developer.lightbend.com/start/?group=akka&amp;project=%s",
       "pekko.doc.dns" -> s"$pekkoBaseURL",
```

Comment:
https://developer.lightbend.com/start/?group=akka&project=%s

?

Reply:
We don't have Pekko versions of all the samples. `"extref.akka-samples.base_url"` has the same value as the previous setting for `"extref.samples.base_url"`. I've changed some of the `extref` settings in the docs to use this `akka-samples` based link - so that the generated link URL remains as before - because the `samples` based links have been changed to link to Pekko URLs if and when there is an actual working Pekko sample to link to.

Reply:
@pjfanning what's the `&amp` in the `akka-sample.base_url`?

Reply:
`&amp;` escapes `&`. This value is not new. I've just changed the extref prefix that maps to it.